config/config.go

```diff
@@ -263,6 +285,86 @@ func (c *toEnable) LogConfig(logger *logrus.Entry) {
 	logger.Info("enabled")
 }
 
+type SourceLoadingConfig struct {
+	Concurrency        uint              `yaml:"concurrency" default:"4"`
+	MaxErrorsPerSource int               `yaml:"maxErrorsPerSource" default:"5"`
+	RefreshPeriod      Duration          `yaml:"refreshPeriod" default:"4h"`
+	StartStrategy      StartStrategyType `yaml:"startStrategy" default:"blocking"`
+	Downloads          DownloaderConfig  `yaml:"downloads"`
```

Comment:
Some of the defaults were changed here.
I don't think we currently have a way to manually write changelog text but if there's an easy way to do it it could be nice to mention this.